azurerm/resource_arm_servicebus_topic.go

```diff
@@ -151,6 +151,19 @@ func resourceArmServiceBusTopicCreate(d *schema.ResourceData, meta interface{})
 		parameters.SBTopicProperties.DuplicateDetectionHistoryTimeWindow = utils.String(duplicateWindow)
 	}
 
+	// We need to retrieve the namespace because Premium namespace works differently from Basic and Standard,
+	// so it needs different rules applied to it.
+	namespacesClient := meta.(*ArmClient).serviceBusNamespacesClient
+	namespace, nsErr := namespacesClient.Get(ctx, resourceGroup, namespaceName)
```

Comment:
Super minor, but can we make nsErr just err to keep it consistent in the codebase?